README.md

```diff
@@ -23,24 +23,28 @@ officially supported languages have well-supported Avro libraries:
 |C++|[Apache Avro C++](https://avro.apache.org/docs/current/api/cpp/html/index.html)|
 |Dart|[avro-dart](https://github.com/sqs/avro-dart) (last updated 2012)|
 |Go|[goavro](https://github.com/linkedin/goavro)|
-|Java|[Apache Avro Java](https://avro.apache.org/docs/1.10.2/gettingstartedjava.html)|
+|Java|[Apache Avro Java](https://avro.apache.org/docs/current/getting-started-java/)|
 |Kotlin|[avro4k](https://github.com/avro-kotlin/avro4k)|
 |Node|[avro-js](https://www.npmjs.com/package/avro-js)|
 |Objective C|[ObjectiveAvro](https://github.com/jlawton/ObjectiveAvro) (but read [this](https://stackoverflow.com/questions/57216446/data-serialisation-in-objective-c-avro-alternative))|
 |PHP|[avro-php](https://github.com/wikimedia/avro-php)|
-|Python|[Apache Avro Python](https://avro.apache.org/docs/current/gettingstartedpython.html)|
+|Python|[Apache Avro Python](https://avro.apache.org/docs/current/getting-started-python/)|
 |Ruby|[AvroTurf](https://github.com/dasch/avro_turf)|
 
 ## Documentation, Blog Post and Videos
 - [Pub/Sub API Developer Guide](https://developer.salesforce.com/docs/platform/pub-sub-api/overview)
 - [Salesforce Architects Blog Post](https://medium.com/salesforce-architects/announcing-pub-sub-api-generally-available-3980c9eaf0b7)
 - [Introducing the New gRPC-based Pub Sub API YouTube Developer Quick Takes](https://youtu.be/g9P87_loVVA)
 
-## Code Samples
+## Salesforce Support Code Samples
```

Comment:
I would replace with "Code Samples from Salesforce"

Reply:
Also add an intro paragraph below this title with this text:

"Salesforce provides these samples for demonstration purposes. They aren't meant to be used in production code. Before you use these samples in production, make sure you perform thorough functional and performance testing."